lib/elixir/lib/base.ex

```diff
@@ -319,13 +319,84 @@ defmodule Base do
             "Double check your string for unwanted characters or pad it accordingly"
   end
 
+  @doc """
+  Checks if a string is a valid base 16 encoded string.
+
+  Use this function when you just need to *validate* that a string is valid base64 data,
```

Comment:
```suggestion
  Use this function when you just need to *validate* that a string is valid base16 data,
```

Also, I think this description is very useful and should be copied to all valid_ functions.
I can otherwise imagine people misunderstanding the intent and going for something like:

```elixir
if Base.valid_decode16?(s), do: Base.decode16!(s)
```

WDYT?

Reply:
Yes good call.

Reply:
Done.